packages/hoppscotch-sh-admin/src/composables/useClientHandler.ts

```diff
@@ -31,9 +32,12 @@ export function useClientHandler<
         })
         .toPromise();
 
-      const resultList = getList(result.data!);
-
-      list.value.push(...resultList);
+      if (getList) {
+        const resultList = getList(result.data!);
+        dataAsList.value.push(...resultList);
```

Comment:
There's a type error arising here; might need an assertion.

Reply:
@jamesgeorge007 When pushing data into dataAsList, the value is of type `UnWrapRefSimple<ListItem>`.  However I am not able to import this type. I am able to import `UnwrapRef` from vue. I am not able to resolve this using assertion. Can you suggest a way to fix the type inconsistency?

Reply:
We'd require an assertion as `Ref<ListItem[]>` while initializing `dataAsList` in this case - Ref: https://github.com/vuejs/core/issues/2136.

Thought to specify the type directly in https://github.com/hoppscotch/hoppscotch/pull/3686/commits/c2c232add299785bdbe0c2842af3b175dfcd9f00 since the assertion was felt redundant.